docs/tutorials/abn/abn.md

```diff
@@ -2,173 +2,185 @@
 
     === "node"
         ```shell
-        kubectl create deployment frontend --image=iter8/abn-sample-frontend-node:0.13
+        # kubectl create deployment frontend --image=iter8/abn-sample-frontend-node:0.15
+        kubectl create deployment frontend --image=kalantar/frontend-node:20230717-1552
         kubectl expose deployment frontend --name=frontend --port=8090
         ```
 
     === "Go"
         ```shell
-        kubectl create deployment frontend --image=iter8/abn-sample-frontend-go:0.13
+        # kubectl create deployment frontend --image=iter8/abn-sample-frontend-go:0.15
+        kubectl create deployment frontend --image=kalantar/frontend-go:20230717-1339
         kubectl expose deployment frontend --name=frontend --port=8090
         ```
     
-    The frontend component is implemented to call *Lookup()* before each call to the backend component. The frontend componet uses the returned track identifier to route the request to a version of the backend component.
+    The frontend component is implemented to call `Lookup()` before each call to the backend component. The frontend componet uses the returned version number to route the request to the recommended version of the backend component.
 
 === "backend"
-    Deploy version *v1* of the *backend* component, associating it with the track identifier *backend*.
+    Deploy an initial version of the *backend* component:
 
     ```shell
     kubectl create deployment backend --image=iter8/abn-sample-backend:0.13-v1
-    kubectl label deployment backend app.kubernetes.io/version=v1
+    kubectl label deployment backend iter8.tools/watch="true"
 
     kubectl expose deployment backend --name=backend --port=8091
     ```
 
+## Describe the Application
+
+In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A `ConfigMap` is used to describe the make up of possible versions:
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: backend
+  labels:
+    app.kubernetes.io/managed-by: iter8
+    iter8.tools/kind: routemap
+    iter8.tools/version: "v0.15"
+immutable: true
+data:
+  strSpec: |
+    versions:
+    - resources:
+      - gvrShort: svc
+        name: backend
+        namespace: default
+      - gvrShort: deploy
+        name: backend
+        namespace: default
+    - resources:
+      - gvrShort: svc
+        name: backend-candidate-1
+        namespace: default
+      - gvrShort: deploy
+        name: backend-candidate-1
+        namespace: default
+EOF
+```
+
+In this definition, each version of the application is composed of a `Service` and a `Deployment`. In the primary version, both named `backend`. In any candidate version they are named `backend-candidate-1`. Iter8 uses this definition to identify when any of the versions of the application are available. It can then respond appropriate to `Lookup()` requests. 
```

Comment:
```suggestion
In this definition, each version of the application is composed of a `Service` and a `Deployment`. In the primary version, both are named `backend`. In any candidate version they are named `backend-candidate-1`. Iter8 uses this definition to identify when any of the versions of the application are available. It can then respond appropriate to `Lookup()` requests. 
```